### ISSUE TITLE
u-boot ethernet phy regression on gxbb (nanopi-k2, odroid-c2)

### ISSUE THREAD

**Issue by sarnold** [superna9999/meta-meson#106]
**Describe the bug**
According to the feedback on this bug: https://github.com/home-assistant/operating-system/issues/1202 ethernet/phy is working at least through 2020.04 u-boot release.  Confirmed broken using poky/meta-meson on dunfell branches.

**Build Context**
 - meta-meson dunfell
 - meta-meson changes: none prior to reverting u-boot-meson-gx version requirement to 2020.01 (u-boot version on poky dunfell)
 - Yocto Project Poky/meta dunfell
 - meta-small-arm-extra dunfell
 - local.conf normal except for:

```
PREFERRED_PROVIDER_virtual/kernel ?= "linux-mainline"
PREFERRED_VERSION_linux-mainline ?= "5.10%"
# ethernet bug somewhere after this version
PREFERRED_VERSION_u-boot-meson-gx = "2020.01%"  <= broken without this
```

**Expected behavior**
Working ethernet.

**System (please complete the following information):**
 - Host OS: gentoo
 - Version: N/A
 - Architecture: x86_64

**Additional context**
Just building core-image-minimal with some packages, testing on nanopi-k2 and odroid-c2 (confirmed working with u-boot 2020.04 and kernel 5.10.14 with one small espressobin-ultra patch).

**Comment by superna9999** [comment]
Fixed with 1a95da2a8bb9bf268551805655f2a3285243971d (upgrade mainline u-boot to 2020.10) on dunfell

**Comment by sarnold** [comment]
Thanks!
